app/scripts/app/models/variable_exploration/available_methods.component.js

```diff
@@ -1,98 +1,102 @@
 "use strict";
-angular
-  .module("chuvApp.models")
-  .component("availableMethods", {
-    bindings: {
-      getFocusedVariable: '<',
-      zoom: '='
-    },
-    require: {
-    },
-    controllerAs: "avmethods",
-    controller:[
-      'Variable',
-      "MLUtils",
-      function(Variable, MLUtils){
-        var vm = this;
+angular.module("chuvApp.models").component("availableMethods", {
+  bindings: {
+    getFocusedVariable: "<",
+    zoom: "="
+  },
+  require: {},
+  controllerAs: "avmethods",
+  controller: [
+    "$scope",
+    "Variable",
+    "MLUtils",
+    "Config",
+    function($scope, Variable, MLUtils, Config) {
+      var vm = this;
 
-        var ml_all_methods = [];
+      var ml_all_methods = [];
 
-        vm.ml_methods = [];
-        vm.$onChanges = () => {
-          // Get all the ml methods
-          MLUtils.list_ml_methods().then(function(data) {
+      vm.ml_methods = [];
+      vm.$onChanges = () => {
+        // Get all the ml methods
+        Config.then(config => config.mode === "local")
+          .then(isLocal => {
+            $scope.isLocal = isLocal;
+          })
+          .then(() => MLUtils.list_ml_methods())
+          .then(function(data) {
             // FIXME Quick and dirty fix!
-            ml_all_methods = data.filter(function(m) {
-              return m.code !== "glm_exareme";
-            });
+            if (!$scope.isLocal) {
```

Comment:
Le flag "isLocal" c'est la seule instance d utilisation de $scope, tu peux le remplacer par this dans le controller et $ctrl dans le template